spec/factories/form526_submissions.rb

```diff
@@ -20,6 +20,12 @@
     end
   end
 
+  trait :hypertension_claim_for_increase do
+    form_json do
+      File.read("#{::Rails.root}/spec/support/disability_compensation_form/submissions/only_526_hypertension.json")
+    end
+  end
+
```

Comment:
Creating `hypertension_claim_for_increase` submissions is now easier and more readable in tests.

Reply:
I didn't realize we had factories setup — great!